### DISCUSSION TITLE
Welcome to CocoaMQTT Discussions!

### DISCUSSION THREAD

**Discussion by JaylinYu** [emqtt/CocoaMQTT#519] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ltAldoRaine** [comment]
Hello,

Is there any chance that Cocoamqtt will use starscream 4.0.0 or greater ?

I have other dependency which is using starscream 4.0.4 and because of that i can't add CocoaMQTT in my project.

P.s I'm using swift package manager.

**Reply by leeway1208** [reply]
Hello. I will upgrade starscream in the future version. Thank you for your support.
